api/http-request-practice.md

```diff
@@ -162,45 +133,10 @@ What happens?
 
 To keep things simple, for this example, we just load the same origin, worldtimeapi.org, so we didn't have to worry about it.
 
-### `http.request` in Node
-
-There are too many HTTP libraries to count in Node, but there is one that is built-in that we'll use.
-
-```javascript
-const options = {
-  hostname: 'worldtimeapi.org',
-  port: 443,
-  path: '/api/ip',
-  method: 'GET'
-};
-
-const req = https.request(options, res => {
-  console.log(`statusCode: ${res.statusCode}`)
-
-  res.on('data', d => {
-    process.stdout.write(d)
-  })
-});
-
-req.end();
-
-```
-
-### Guided Practice
-
-`fetch` is a standard added by browsers but if you like it, it's also available as a [package for node](https://www.npmjs.com/package/node-fetch). Let's use it!
-
-- `npm install --save node-fetch`
-- open node REPL
-- `require('node-fetch')`
-- Run the same command from fetch in the browser but in your node repl
-
-
 ### Independent Practice
 
 Since there are tons of libraries out there, now's a great time to try a few and see what you like/dislike about them compared to those we used above.
 
-- [5 Ways to Make HTTP Requests in Node.js](https://www.twilio.com/blog/2017/08/http-requests-in-node-js.html)
```

Comment:
Why remove this?

Reply:
Remove these sections
XMLHttpRequest
http.request in Node.js
fetch in node

Reply:
It is written in the issue to remove the parts. Pl see #1199

Reply:
I disagree, pleas put it back. @vegetabill let me know if this is a problem.